airbyte-integrations/connectors/source-mysql/src/main/resources/spec.json

```diff
@@ -54,11 +54,140 @@
         "default": true,
         "order": 6
       },
+      "ssl_mode": {
+        "title": "SSL modes",
+        "description": "SSL connection modes. \n <ul><li><b>disable</b> - Use to make a general connection without SSL.</li>\n <li><b>preferred</b> - Automatically attempt SSL connection. If the MySQL server does not support SSL, continue with a regular connection.</li>\n <li><b>required</b> - Always connect with SSL. If the MySQL server doesn’t support SSL, the connection will not be established. Certificate Authority (CA) and Hostname are not verified.</li>\n  <li><b>verify-ca</b> - Always connect with SSL. Verifies CA, but allows connection even if Hostname does not match.</li>\n  <li><b>Verify Identity</b> -Always connect with SSL. Verify both CA and Hostname.</li></ul>\n Read more <a href=\"https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-reference-using-ssl.html\"> in the docs</a>.",
+        "type": "object",
+        "order": 7,
+        "oneOf": [
+          {
+            "title": "disable",
```

Comment:
yes, you are right, it does break the logic a bit. It would be nice to remove the SSL enable/disable toggle and leave only the SSL modes dropdown list, but that would break backwards compatibility of connector versions. Therefore, I agree with you that it is better to remove the "disable" value. Did it.